src/interface_r/scripts/gen_wrappers.R

```diff
@@ -13,8 +13,8 @@ pkg_location <- "~/h2o4gpu/src/interface_r"
 
 
 # Note: Code below should not be changed unless necessary
-
-source(file.path(pkg_location, "scripts/gen_wrapper_utils.R"))
+setwd(pkg_location)
```

Comment:
This is needed because the file names are all based on it. I think it's safe to change working directory to this (also makes things easier for development).